### ISSUE TITLE
MacosTextField placeholder has too low contrast ratio

### ISSUE THREAD

**Issue by stephane-archer** [macosui/macos_ui#566]
<img width="1440" alt="Image" src="https://github.com/user-attachments/assets/e6782212-2d64-4458-991c-c6c0ff5f5c97" />

```  Dart
Container(
                    color: userConvertionParameters.destWidth == null ||
                            userConvertionParameters.destWidth == 0
                        ? darkRed
                        : darkGreen,
                    child: MacosTextField(
                      controller: _widthController,
                      placeholder: "Width: (Example: \"1280\")",
                      keyboardType: TextInputType.number,
                      onChanged: (String value) {
                      },
                    )
``` 
the placeholder on MacosTextField is too dark for most people, there should be a better contrast ratio

**Comment by Vorkytaka** [comment]
Perhaps the problem is somewhere between Flutter and macos_ui.
There is everything OK with Flutter 3.24.0, but got the same bug with 3.29.2.

**Comment by stephane-archer** [comment]
Why close the issue ? Is it solved ?

**Comment by GroovinChip** [comment]
It was closed because of the comment noting the issue was not present on a more recent version of Flutter. If you are still encountering this issue on the most recent versions of Flutter and `macos_ui` we can reopen the issue.

**Comment by stephane-archer** [comment]
@GroovinChip 3.24.0 < 3.29.2

**Comment by GroovinChip** [comment]
> @GroovinChip 3.24.0 < 3.29.2

This the Flutter version you're seeing this with? What about the `macos_ui` version?

**Comment by stephane-archer** [comment]
Can you just reproduce with the code snippet on the latest version and a dark theme ?
